server/settings/simple_creators.py

```diff
@@ -2,7 +2,7 @@
 
 
 class SimpleCreatorPlugin(BaseSettingsModel):
-    _layout = "expanded"
```

Comment:
This really should stayed there.

Reply:
IMO, Expanded layout in this setting  is very confusing. it's very hard for me to go through it.
![image](https://github.com/user-attachments/assets/91677078-204b-4bb9-a45c-da330d933899)

Reply:
I tried updating the setting group name with `label` but that was not recommended because admins are allowed to leave the label blank. 

Also, I tried updating the setting group name with `product_type` but AYON server ignored the name in favor of the label. 
https://discord.com/channels/517362899170230292/773127690453516298/1313448327903445012

Reply:
I know you've tried to change, but then reverted, so this should be reverted too. It was bad before, now it is worse...

Reply:
I'll create another PR this afternoon.